auth/genericoauth/provider.go

```diff
@@ -47,6 +50,7 @@ type GenericOAuthConfig struct {
 	AuthURLParams map[string]string `json:"auth_url_params,omitempty"   long:"auth-url-param"  description:"Parameter to pass to the authentication server AuthURL. Can be specified multiple times."`
 	Scope         string            `json:"scope,omitempty"             long:"scope"           description:"Optional scope required to authorize user"`
 	TokenURL      string            `json:"token_url,omitempty"         long:"token-url"       description:"Generic OAuth provider TokenURL endpoint."`
+	CACert        string            `json:"ca_cert,omitempty"           long:"ca-cert"         description:"PEM-encoded CA certificate string"`
```

Comment:
Could you make this a path to a file containing the cert instead? That would be consistent with the rest, e.g. https://github.com/concourse/atc/blob/master/auth/uaa/provider.go#L57

Reply:
done